submarine-commons/commons-unixusersync/src/test/java/org/apache/submarine/commons/unixusersync/jwtToken.java

```diff
@@ -0,0 +1,81 @@
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.submarine.commons.unixusersync;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwtBuilder;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.impl.crypto.MacProvider;
+import jdk.internal.jline.internal.Log;
+
+import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
+import java.security.Key;
+import java.util.Base64;
+import java.util.Date;
+
+public class jwtToken {
+
+  private static final Key secret = MacProvider.generateKey(SignatureAlgorithm.HS256);
+  private static final byte[] secretBytes = secret.getEncoded();
+  private static final String base64SecretBytes = Base64.getEncoder().encodeToString(secretBytes);
+
+  //Sample method to construct a JWT
+
+  public static String createJWT(String id, String issuer, String subject, long ttlMillis) {
+  //The JWT signature algorithm we will be using to sign the token
+    SignatureAlgorithm signatureAlgorithm = SignatureAlgorithm.HS256;
+
+    long nowMillis = System.currentTimeMillis();
+    Date now = new Date(nowMillis);
+
+  //We will sign our JWT with our ApiKey secret
+    Key signingKey = new SecretKeySpec(secretBytes, signatureAlgorithm.getJcaName());
+
+  //Let's set the JWT Claims
+    JwtBuilder builder = Jwts.builder().setId(id)
+            .setIssuedAt(now)
+            .setSubject(subject)
+            .setIssuer(issuer)
+            .signWith(signatureAlgorithm, signingKey);
+
+  //if it has been specified, let's add the expiration
+    if (ttlMillis >= 0) {
+      long expMillis = nowMillis + ttlMillis;
+      Date exp = new Date(expMillis);
+      builder.setExpiration(exp);
+    }
+
+  //Builds the JWT and serializes it to a compact, URL-safe string
+    return builder.compact();
+  }
+
+  //Sample method to validate and read the JWT
+  public void parseJWT(String jwt) {
+  //This line will throw an exception if it is not a signed JWS (as expected)
+    Claims claims = Jwts.parser()
+        .setSigningKey(DatatypeConverter.parseBase64Binary(base64SecretBytes))
+        .parseClaimsJws(jwt).getBody();
+    Log.info("ID: " + claims.getId());
+    Log.info("Subject: " + claims.getSubject());
+    Log.info("Issuer: " + claims.getIssuer());
+    Log.info("Expiration: " + claims.getExpiration());
```

Comment:
Should use `org.slf4j.Logger` instead of `jdk.internal.jline.internal.Log`.
like., https://github.com/apache/submarine/blob/master/submarine-commons/commons-unixusersync/src/test/java/org/apache/submarine/commons/unixusersync/EmbeddedLdapRuleTest.java#L56